packages/client/src/App.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+// import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
-import { LandingPage } from './containers/LandingPage/LandingPage.Container';
-import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
+import Login from './component/Login';
+// import { LandingPage } from './containers/LandingPage/LandingPage.Container';
```

Comment:
In general it is a good idea to not leave comments in the code that is being committed to the repository, since it can clutter the code.

The benefit of using git is that you can always go back in history and see what lines were removed, etc.